packages/@okta/vuepress-theme-prose/components/TermsAndConditionsDialog.vue

```diff
@@ -0,0 +1,267 @@
+          v-model="selectedRegion"
+        >
+          <option disabled selected></option>
+          <option
+            v-for="regionData in region.list"
+            v-bind:key="regionData.name"
+            :value="regionData.value"
+            >{{ regionData.name }}</option
+          >
+        </select>
+      </label>
+      <div class="marketing-c" v-show="isGDPRCountry">
+        <div class="title">
+          Marketing Communication <span> - Optional</span>
+        </div>
+        <div class="gdpr-condition">
+          <input
+            type="checkbox"
+            name=""
+            id="okta-contact"
+            ref="gdprBoxEl"
+            checked="false"
+          />
+          <label for="okta-contact">
+            I agree that Okta may contact me with marketing communications. See
+            Privacy Policy for details on how to unsubscribe.
+          </label>
+        </div>
+      </div>
+    </div>
+    <template v-slot:footer>
+      <div class="agree-policy" v-show="selectedCountry">
+        <div class="splitter"></div>
+        <p>
+          By clicking "continue", I agree to the applicable Free Trial terms in
+          <SmartLink :item="{ link: '/terms/' }">Okta's Terms</SmartLink> and
+          the processing of my personal data by Okta as described in the
+          <SmartLink :item="{ link: '/privacy-policy/' }"
+            >Privacy Policy</SmartLink
+          >.
+        </p>
+      </div>
+      <div class="terms-conditions-btns">
+        <div class="btn-container">
+          <input
+            type="button"
+            class="btn red-button"
+            @click="setTaCUrlAndRedirect()"
+            :class="{ 'btn-disabled': isDisabledSocialAuth }"
+            value="continue"
+            :disabled="isDisabledSocialAuth"
+          />
+        </div>
+        <div class="btn-container mr-15">
+          <input
+            type="button"
+            class="btn social-btn"
+            value="cancel"
+            @click="closeTermsConditionsDialog()"
+          />
+        </div>
+      </div>
+    </template>
+  </DialogWindow>
+</template>
+
+<script>
+import {
+  countriesList,
+  americanStates,
+  canadaProvinces,
+  GDPR_COUNTRIES
+} from "../const/signup.const";
+
+const CANADA = "Canada";
+const USA = "United States";
+
+export default {
+  name: "TermsAndConditionsDialog",
+  components: {
+    SmartLink: () => import("./SmartLink"),
+    DialogWindow: () => import("./DialogWindow")
+  },
+  props: {
+    socialUrl: {
+      required: true,
+      type: String
+    }
+  },
+  data() {
+    return {
+      selectedCountry: "",
+      selectedRegion: "",
+      regionData: { type: "", list: [] }
+    };
+  },
+  computed: {
+    isDisabledSocialAuth() {
+      // If country not selected, disable "continue" button.
+      if (!this.selectedCountry) {
+        return true;
+      }
+      // If country Canada or USA but satate not selected, disable "continue" button.
```

Comment:
```suggestion
      // If country Canada or USA but state not selected, disable "continue" button.
```

Reply:
yes, this is from the new task https://oktainc.atlassian.net/browse/OKTA-378018